src/lib/data-helper.ts

```diff
@@ -109,16 +110,32 @@ const collectVariableCheckGraderOptions = (container: Cash): VariableCheckGrader
   }
 }
 
-export const collectUnitTest = (container: Cash): UnitTest => {
+export const collectUnitTest = (container: Cash): AssertEqualParams => {
```

Comment:
`collectUnitTest` should collect `UnitTest` not `AssertEqualParams`
with `UnitTest.name`